src/iccpd/src/iccp_cli.c

```diff
@@ -83,8 +83,11 @@ int set_peer_link(int mid, const char* ifname)
     if (csm == NULL)
         return MCLAG_ERROR;
 
-    if (len > IFNAMSIZ)
+    if (len > MAX_L_PORT_NAME)
+    {
+        ICCPD_LOG_ERR(__FUNCTION__, "Peer-link %s, Strlen %d greater than MAX:%d ", ifname, strlen, MAX_L_PORT_NAME);
```

Comment:
Should it be logging "strlen" or "len" instead?

Reply:
@gechiang : Thanks, replaced with 'len'.